packages/admin/cms-admin/src/dam/Table/breadcrumbs/FolderBreadcrumbs.tsx

```diff
@@ -18,14 +18,28 @@ interface DamBreadcrumbItem {
     url: string;
 }
 
-type FolderBreadcrumbProps = DamBreadcrumbItem;
+interface FolderBreadcrumbProps extends DamBreadcrumbItem {
+    overrideLabel?: React.ReactNode;
```

Comment:
You could make label required and leave the "calculation" to the parent components